src/site/views/ticketpurchase/tmpl/summary.php

```diff
@@ -101,30 +140,39 @@
 					$option.val('NULL');
 					$option.prop('disabled', true);
 				}
-
-				$totalTicketPrice();
-				$updateStripeButton(event)
 			};
 
-			$qtySelectors.change(function (event) {
+			$qtySelectors.change(function(event) {
 				$qtyChanged(event);
+				$updateTicketPrice();
+				$determineValidAddon(event)
+				setStripeButtonStatus();
 			});
 
-			$optionSelectors.change(function (event) {
-				$updateStripeButton(event);
+			$optionSelectors.change(function(event) {
+				$determineValidAddon(event);
+				setStripeButtonStatus();
+				console.log($generateAddonsArray());
 			});
 		}
+
+		$updateTicketPrice();
 	});
 </script>
 
+
+<!-- create form -->
+<script src="https://js.stripe.com/v3/"></script>
```

Comment:
Would it be better to load this from the Stripe library we install with the com_swa component rather then getting it from an external site?
Similar to how the Stripe stylesheet is loaded at the top of this file perhaps?

Reply:
https://stripe.com/docs/js/including
seems like it should be loaded from stripe url. Not sure though